scout/server/templates/utils.html

```diff
@@ -33,6 +33,35 @@
   </table>
 {% endmacro %}
 
+
+{% macro variant_related_comments_table(variant, case) %}
+      <table role="presentation" id="panel-table" class="table table-sm" style="background-color: transparent">
+        <thead>
+          <tr>
+            <td colspan=2><strong>Variant-related comments</strong></td>
+          </tr>
+        </thead>
+        <tbody>
+          {% for comment in variant.comments %}
+            <tr>
+              <td style="width:50%;">{{comment.created_at.strftime('%Y-%m-%d')}} {{comment.user_name}}
+                {% if comment.level == 'global' %}
+                  <span class='badge bg-info'>GLOBAL</span>
```

Comment:
```suggestion
                  <span class='badge bg-info text-white'>GLOBAL</span>
```

Reply:
White text is more visible I think!

<img width="242" alt="image" src="https://user-images.githubusercontent.com/28093618/222117188-727ed3df-fa2b-49e6-81a3-76016e9dddf7.png">